components/VM_Arm/plat_include/zynqmp/plat/smc.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2022, DornerWorks
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#define PM_GET_API_VERSION              0xC2000001
```

Comment:
I wonder if these should have an `SCM_` prefix to minimize the risk of getting naming clashes.